README.md

```diff
@@ -2,6 +2,15 @@
 
 Checkout the online version: http://opentechschool.github.io/web-frontend-colearning-metrics
 
+## Running
+
+## Mac
+Run this example with a local web server. For example, for Mac users you can use this command from your terminal
+`$ python -m SimpleHTTPServer`
```

Comment:
This should be a code block, like:

    ```
    $ python -m SimpleHTTPServer
    ```

Also, the wording (why are Mac users an *example* in the Mac section?) is questionable.